CONTRIBUTING.md

```diff
@@ -447,9 +451,9 @@ Setup [rudder-transformer](https://github.com/rudderlabs/rudder-transformer) on
 Understand the code structure
 
 
-* `src/cdk/v2/destinations` - Destination integrations
+* `src/v0/destinations` - Destination integrations
 * `src/v1/sources` - Source integrations
-* `src/v0` and `src/v1` - Older integrations. If you’re fixing bugs in one of the older integrations, you might find your integration in these folders if it is an older integration.
+* `src/v0` and `src/v1` - Older integrations. If you're fixing bugs in one of the older integrations, you might find your integration in these folders if it is an older integration.
```

Comment:
you can remove this complete line now. Just mention about `src/v1/destinations` as they are used for network handlers